src/main/java/com/automationexercise/Pages/TestCasesPage.java

```diff
@@ -0,0 +1,24 @@
+package com.automationexercise.Pages;
+
+import com.automationexercise.DriverManager.BrowserFactory;
+import com.automationexercise.Utilities.WaitsUtils;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import static com.automationexercise.Utilities.Utility.verifyElementVisible;
+
+public class TestCasesPage {
+    private final WebDriver driver;
+    private final By verification_ele =  By.xpath("//*[@id=\"form\"]/div/div[2]/h5/span");
+
+    public TestCasesPage(WebDriver driver) {
+        this.driver = driver;
+    }
+    public boolean isTestCasesPageVisible() {
+        verifyElementVisible(verification_ele);
```

Comment:
return  **verifyElementVisible(verification_ele);** itself cause it's a boolean function. and you current implementation will always return true even the element is not visible.